tests/common/widgets/current-refinements/options.ts

```diff
@@ -24,9 +24,14 @@ export function createOptionsTests(
 ) {
   describe('options', () => {
     const searchClient = createAlgoliaSearchClient({});
+    const onSubmitListener = jest.fn();
 
     beforeEach(() => {
       searchClient.search.mockClear();
+
+      const form = document.createElement('form')!;
+      form.addEventListener('submit', onSubmitListener);
```

Comment:
```suggestion
      body.addEventListener('submit', onSubmitListener);
```

and like this we won't need to make anything conditional

Reply:
I'm afraid this lead to false positive in the test of "not calling the event handler". Also, wouldn't the buttons need to be inside the form for the body onSubmit handler to catch the event? In this case it would still require the root container of the widgets to be set as the form element.

Reply:
I see what you mean, but it could also be false positive now if in the setup the button isn't put in the form but directly in the body. We can solve both by adding a test before the assertion that the layout is `body > form > button`